pyaptamer/datasets/data/dummy_data.csv

```diff
@@ -0,0 +1,2 @@
+aptamer,protein,label
```

Comment:
What do you think about structuring the `data` directory and moving files into their respective paths? SO `pdb` goes into `data/pdb` and so on. @fkiraly thoughts?
ps: If you agree and do this, you will have to change the paths of the helper functions too that I added in utils for file conversions.